pkg/containerspec/containerspec_test.go

```diff
@@ -482,3 +482,53 @@ func TestImpliedEqualCPULimit(t *testing.T) {
 		},
 	)
 }
+
+
+func TestCmp_QuantityComparingToIsEqual_ReturnZero(t *testing.T) {
+	kubeResourceQuantity := KubeResourceQuantity("10")
+	kubeResourceQuantityToCompare := KubeResourceQuantity("10")
+
+	result, err := kubeResourceQuantity.Cmp(kubeResourceQuantityToCompare)
+	if err != nil {
+		t.Errorf("Unexpected error, %s", err.Error())
+	}
+
+	expected := 0
+
+	if !assert.Equal(t, expected, result) {
```

Comment:
if we do `assert.Equal` and the assertion fails, the test case fails, right? Am I missing sth? :)

Reply:
If the expected and equal, are not equal
assert.Equal(t, expected, result)
Then it returns false

if False

So never goes in if statement

But we want it to fail the test if not equal, so add the '!'

Looked at using assert.NotEqual, but that causes test to fail when are equal